custom/icds_reports/utils/__init__.py

```diff
@@ -185,11 +184,6 @@ def custom_data(self, selected_location, domain):
         tags = ["location_type:{}".format(loc_type), "report_slug:{}".format(self.slug)]
         if self.allow_conditional_agg:
             tags.append("allow_conditional_agg:yes")
-        datadog_histogram(
-            "commcare.icds.block_reports.custom_data_duration",
-            timer.duration,
-            tags=tags
-        )
```

Comment:
not referenced in any DD dashboard. @sravfeyn are these still in use at all?

Reply:
Not really, it was added when doing performance improvements.